apps/dotcom/sync-worker/src/TLUserDurableObject.ts

```diff
@@ -50,15 +50,21 @@ export class TLUserDurableObject extends DurableObject<Environment> {
 
 	userId: string | null = null
 
+	lastReplicatorBootId: string | null = null
+
 	readonly router = Router()
 		.all('/app/:userId/*', async (req) => {
 			if (!this.userId) {
 				this.userId = req.params.userId
 			}
-			await this.ctx.blockConcurrencyWhile(async () => {
-				this.store.initialize(await this.replicator.fetchDataForUser(this.userId!))
-				this.replicator.registerUser(this.userId!)
-			})
+			if (this.lastReplicatorBootId === null) {
+				await this.ctx.blockConcurrencyWhile(async () => {
+					const { data, bootId } = await this.replicator.fetchDataForUser(this.userId!)
+					this.lastReplicatorBootId = bootId
+					this.store.initialize(data)
+					this.replicator.registerUser(this.userId!)
+				})
+			}
```

Comment:
now that we have the bootId thing, we can avoid fetching the initial data on every connect since we are guaranteed it will be consistent